unit_tests/sources/declarative/auth/test_oauth.py

```diff
@@ -301,6 +301,7 @@ def test_initialize_declarative_oauth_with_token_expiry_date_as_timestamp(
             client_id="{{ config['client_id'] }}",
             client_secret="{{ config['client_secret'] }}",
             token_expiry_date=timestamp,
+            access_token_value="some_access_token",
```

Comment:
I had to change this because if the access_token is not initialized, the `get_token_expiry_date ` will be datetime.min to force a refresh.

The use case in this test didn't make sense to me as there were no access token but there was an expiry date. I'm not sure in which case this is possible.